src/meili/map_matcher.cc

```diff
@@ -378,7 +378,7 @@ MapMatcher::AppendMeasurement(const Measurement& measurement)
   }
   const auto& candidates = candidatequery_.Query(
       measurement.lnglat(),
-      measurement.sq_search_radius(),
+      std::max(measurement.sq_search_radius(), measurement.sq_gps_accuracy()),
```

Comment:
the trick is we dont currently limit gps accuracy so we should do so to make sure this doesnt get too large